### DISCUSSION TITLE
Mac client will not start.

### DISCUSSION THREAD

**Discussion by prozelcs** [rustdesk/rustdesk-server-pro#229] [Q&A]
Hello.

In our company, now that there is the option to generate the Mac client, we are generating the clients. The first test, when trying to start some client generated by the generator, we get that it can not open the application.

Mac version:
Sonom 14.4.1 (23E224)

Error:

handle LS launch error: {\n    AppPath = "/Applications/Suport.app";\n    ErrorCode = "-10810";\n}

Greetings.

**Comment by rustdesk** [comment]
Wondering if this http://thexlab.com/faqs/error-10810.html can help you.

**Reply by rustdesk** [reply]
Or if reboot can help you, https://github.com/beeware/briefcase/issues/66#issuecomment-617481222
